### PR TITLE
final-touches-pullRequest

### LINKED ISSUE #6: final-touches
### Issue: final-touches
- This part of the project is about giving the style to our leaderboard list app.
- We will use plain CSS 
- We will respect the wireframe given 
- We will use Stylelint tool to let our codebase clean and organized.

### Goal: 
The goal is to use plain CSS to give the style to our leaderboard list app.
Besides, we need to respect the wireframe given.

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
### About this project
- This part of the leaderboard list app is to add the styles.
-  For it : we created a feature branch
-  The stylesheet is created using 100%  plain CSS. 
-  The code was tested using Stylelint linter analize tool.
-  The given wireframe is respected.
-  The Best practices of CSS and HTML were followed.

### Goal
The goal of the project is to understand how to build a Game, where this process begin, which methods we need to use, what dependencies we need to install, how to connect the API, we need to work respecting the Gitflow, and finally we have to consider we need to use the best practices of CSS and HTML as well as JS to develop this project.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
    <header>
-        <h1>Leaderboard</h1>
+        <h1>⭐️🌸😊 Welcome to my Leaderboard 😊🌸⭐️</h1>
    </header>
    <section class="main">
         <article class="score">
             <div class="info">
-                <h2 class="title">Recent scores</h2>
+                <h2 class="title">Recent scores 🥇🎖🏆</h2>
                 <button type="button" class="btn-refresh">Refresh</button>
```
Comment (Johnadibe) [inline src/index.html]: > - [ ] Impressive job on the aesthetically pleasing app styling!👍 However, I've noticed that the refresh button is not aligned with the recent score text and also the "submit" `button` is not aligned properly with the provided design. I suggest adjusting the styling of the recent scores and the submit button to match the provided wireframe design. This attention to detail will enhance the overall visual consistency.😊
> 
> Provided design wireframe:
> 
> red
> 
> Your design:
> 
> red1
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @IvonneBenitesRodriguez👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used Git Flow correctly 👍🏽
✅ Linter checks passed✔️
✅ Nice design✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (IvonneBenitesRodriguez) [reply src/index.html]: Hello @Johnadibe 👋🏾
Very nice to meet you
Thank you for the suggestions, I appreciate it,
I modified it according to what you suggested,
thank you again,
Best wishes,
Ivonne

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @IvonneBenitesRodriguez 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great
-  ✅ Your README looks good
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**